gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/core/documentation/use_case/ApiUpdateDocumentationPageUseCaseTest.java

```diff
@@ -111,7 +111,8 @@ class ApiUpdateDocumentationPageUseCaseTest {
     private final DocumentationValidationDomainService documentationValidationDomainService = new DocumentationValidationDomainService(
         new HtmlSanitizerImpl(),
         new NoopTemplateResolverDomainService(),
-        apiCrudService
+        apiCrudService,
+        pageCrudService
     );
     AuditCrudServiceInMemory auditCrudService = new AuditCrudServiceInMemory();
     UserCrudServiceInMemory userCrudService = new UserCrudServiceInMemory();
```

Comment:
Same here --> Add tests for business logic of updating swagger page + async api page.